### DISCUSSION TITLE
Welcome to VexFlow Discussions!

### DISCUSSION THREAD

**Discussion by 0xfe** [0xfe/vexflow#820] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mscuthbert** [comment]
Hi everyone -- I'm Myke Cuthbert -- prof. at MIT in computational music [analysis/musicology/etc.] and also co-founder of Artusi Inc., which uses Vexflow extensively.  I'm a small contributor to Vexflow but been with it for a long time.  I also make the music21j toolkit which is a way of manipulating scores that can redraw vexflow for new changes rather quickly.

**Reply by 0xfe** [reply]
Hi Myke! 👋

**Reply by rvilarl** [reply]
Hi Myke!

**Comment by rvilarl** [comment]
Hi everyone! My name is Rodrigo Vilar. Why I am here? I always wanted to play piano. Now with Corona, I decided to buy a Yamaha P-125 which came with a flowkey subscription (www.flowkey.com) I did like the concept MIDI Piano / App helping to learn songs. I also did like the amount of MusicXML scores in the net. I suggested the flowteam to allow loading these scores and they did not like the idea so I built it myself (www.michaelecke.com/pianoplay) This idea took me to OpenSheetMusicDisplay and VexFlow. I like both projects a lot and I am happy to collaborate where I can.

**Reply by mscuthbert** [reply]
Hi Rodrigo!  Cool app!

**Reply by rvilarl** [reply]
Thank you!

**Comment by ronyeh** [comment]
Sorry I'm late to this thread. I feel like we're all best friends already! :-) 

My name's Ron Yeh. I grew up in California, studied computer science at UC Berkeley, and then spent a LOT of time surfing the web at Stanford (while studying human-computer interaction & computer graphics). Eventually, they gave me a PhD and shooed me out of my office.

I'm learning how to play piano and guitar...... so instead of practicing, of course I decided I should build a tool to help myself learn music faster. Ha. That's how I discovered VexFlow.

It's been fun working with you all so far. Hopefully one day we'll get to meet up (in person! 😮).

**Reply by 0xfe** [reply]
A very belated welcome! :-)

**Reply by AaronDavidNewman** [reply]
>> Hopefully one day we'll get to meet up (in person! 😮).

VexFlowCon 2022!
